pages/linux/umask.md

```diff
@@ -0,0 +1,15 @@
+# umask
+
+> Display file creation mask or set file creation mask to octal value.
```

Comment:
This description wasn't entirely clear for me. Wikipedia puts it better: "determines the settings of a **mask** that controls how file permissions are set for [the current **user**'s] newly created files" (emphasis mine). Shall we rewrite the command description to use something closer to this?